Arquivos/indexada.py

```diff
@@ -6,14 +6,15 @@
     info = "\nuser@os:~"
     text = info + "r/$ "
     command = ""
-    opsystem = OS(memory_size=512)
+    opsystem = OS(memory_size=20)
 
     while command != "exit":
         command = input(text)
 
         inputs = command.split(" ")
 
-        if len(inputs) == 1 and inputs[0] != "ls":
+        if (len(inputs) == 1 and inputs[0] != "ls" and inputs[0] != "info"
+            and inputs[0] != "currinfo" and inputs[0] != "help"):
```

Comment:
melhor fazer uma lista com esses comandos:
`comandos_aceitos = ["ls", "info", "currinfo", "help"]`
e fazer um:
```suggestion
            if len(inputs) == 1 and inputs[0] not in comandos_validos:
```